src/pages/contacts.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from "@emotion/styled";
+import { Logo } from "../components/Logo";
+import { Menu } from "../components/Menu";
+import { PageProps } from "gatsby";
+import { Button, TextField } from "@mui/material";
```

Comment:
I'm not sure that using mui is a good idea
can you check size of bundle before your pr and after?
there is some special tools for it, right now I don't have good internet to find it

Reply:
@jathpr i didn't install it, it already exists in the project. Menu component is created with mui

Reply:
yes, but size of bundle grow up with each component that you use
you can read about tree shaking for example